docs/network-flow-visibility.md

```diff
@@ -1,12 +1,97 @@
-# ELK Flow Collector
-## Purpose
-Antrea supports sending IPFIX flow records through a flow exporter. The Elastic
+# Network Flow Visibility in Antrea
+
+## Table of Contents
+- [Overview](#overview)
+- [Flow Exporter feature](#flow-exporter-feature)
+  * [Configuration](#configuration)
+  * [Supported capabilities](#supported-capabilities)
+- [ELK Flow Collector](#elk-flow-collector)
+  * [Purpose](#purpose)
+  * [About Elastic Stack](#about-elastic-stack)
+  * [Deployment Steps](#deployment-steps)
+  * [Pre-built Dashboards](#pre-built-dashboards)
+    + [Overview](#overview-1)
+    + [Flows](#flows)
+      - [Pod-to-Pod Traffic](#pod-to-pod-traffic)
+      - [Pod-to-Service Traffic](#pod-to-service-traffic)
+    + [Flow Records](#flow-records)
+    + [Node Throughput](#node-throughput)
+      
+## Overview
+[Antrea](architecture.md) is a Kubernetes network plugin that provides network connectivity and security features for Pod workloads.
+Considering the scale and dynamism of Kubernetes workloads in a cluster, Network Flow Visibility helps in the management
+and configuration of Kubernetes resources such as Network Policy, Services, Pods etc., and thereby provides opportunities
+to enhance the performance and security aspects of Pod workloads.
+
+For visualizing the network flows, Antrea monitors the flows in Linux conntrack module. These flows are converted to flow records
+and are sent to the configured flow controller. High-level design is given below:
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/netviz.png" width="600" alt="Network Flow Visibilty">
+
+## Flow Exporter feature
+
+In Antrea, the basic building block for the Network Flow Visibility is the **Flow Exporter feature**. Flow Exporter operates
+within Antrea Agent; it builds and maintains a connection store by polling and dumping flows from conntrack module periodically.
+The connections from connection store are exported to a flow collector using IPFIX protocol, and for this purpose we
```

Comment:
s/The connections from connection store are exported to a flow collector using IPFIX protocol/Connections from the connection store are exported to a flow collector using the IPFIX protocol